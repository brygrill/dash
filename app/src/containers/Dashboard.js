/* eslint-disable react/prefer-stateless-function */
import React, { Component } from 'react';

class Dashboard extends Component {
  constructor() {
    super();
    this.state = {
      user: null,
    };
  }

  render() {
    return (
      <div>
        <h1>Dashboard!</h1>
      </div>
    );
  }
}

export default Dashboard;
