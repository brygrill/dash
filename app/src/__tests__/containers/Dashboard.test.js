import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Dashboard from '../../containers/Dashboard';

it('Dashboard container renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Dashboard />, div);
});

test('Dashboard shows dashboard page!', () => {
  const component = renderer.create(<Dashboard />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
