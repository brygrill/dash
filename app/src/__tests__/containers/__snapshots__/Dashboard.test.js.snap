exports[`test Dashboard shows dashboard page! 1`] = `
<div>
  <h1>
    Dashboard!
  </h1>
</div>
`;
